### PR TITLE
Parth/flask commands

### PR DESCRIPTION
**Pull request by gupta-parth**
Changed the clear_database.py file to commands.py file so that we can add additional custom commands in the future if need be. Also changed the blueprint name to `commands`.

Usage:
To clear the database,
`flask commands cleardb`

### CODE REVIEW

File: software/speller/data_collection_platform/backend/dcp/__init__.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def create_app():
         migrate.init_app(app, db)
 
         # Added blueprint for clearing the database
-        from .clear_database import usersbp
-        app.register_blueprint(usersbp)
+        from .commands import commands_bp
```
Comment (mlej8) [inline software/speller/data_collection_platform/backend/dcp/__init__.py:46]: let's not use relative imports here, 
```suggestion
        from dcp.commands import commands_bp
```

File: software/speller/data_collection_platform/backend/dcp/commands.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+from distutils import command
```
Comment (mlej8) [inline software/speller/data_collection_platform/backend/dcp/commands.py:1]: what is this is it tested?

**Comment by mlej8** [comment]
lgtm @gupta-parth, squash and merge the feature once the you are done reviewing the comments ^

**Comment by gupta-parth** [comment]
Thanks for the suggestions, code has been updated.

**Pull request merged**